podcasts/VideoViewController.swift

```diff
@@ -90,7 +90,11 @@ class VideoViewController: SimpleNotificationsViewController, AVPictureInPicture
     @IBOutlet var pipButton: UIButton!
 
     @IBOutlet var airplayButton: UIButton!
+    #if APPCLIP
+    @IBOutlet var castButton: UIButton!
```

Comment:
Does it make sense to keep the cast button on the App clip? Will it work?

Reply:
This needs to remain because the xib sets it and it will crash otherwise. It isn't used in the App Clip, though.

You can test the video playback by changing the Invocation URL to `https://pca.st/q319mb8y` in the Run Arguments of the App Clip target:
![CleanShot 2024-12-19 at 16 39 39@2x](https://github.com/user-attachments/assets/4053fff2-3b6d-40ed-8f17-bd6f758c17d0)